src/module.c

```diff
@@ -190,6 +190,11 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
 		return REDISMODULE_ERR;
 	}
 
+	if (RedisModule_CreateCommand(ctx, "graph.INFO", Graph_Info, "readonly", 1, 1,
+								1) == REDISMODULE_ERR) {
+		return REDISMODULE_ERR;
```

Comment:
`GRAPH.INFO` is expected to produce information about a specified graph (correct me if I'm wrong)
If so, then the description on the numeric overflow PR: "This is used by GRAPH.INFO when collecting information from multiple shards" is a bit misleading as it hints for data collection from multiple shards. (a graph resides on a single shard)